### ISSUE TITLE
Show on map Not working

### ISSUE THREAD

**Issue by moorebi** [esm7/obsidian-map-view#73]
when I select "Show On Map" I get an error message saying "Map View unable to open map titles.  Try switching the map source using the View controls".  Not sure where to go next.. I have looked in the documentation and come to dead ends.. any thoughts on how to fix.. I can click on same link and select "Open with Google Maps" and it works to open in a new tab in Chrome..

**Comment by esm7** [comment]
1. In the map controls, click View and choose the map source CartoDB, this should work.
2. If it doesn't, can you open the Obsidian Developer Console (Ctrl+Shift+I on PC) and copy or screenshot what you see there when the error happens?

**Comment by moorebi** [comment]
Yes Click on that "CartoDB" and still get error.. here is the screen shot from Console.. let me know if you need something else.. 

![image](https://user-images.githubusercontent.com/6247939/166060193-4783a2e8-b4c0-41ab-8cf2-6c29bdbba471.png)

**Comment by esm7** [comment]
These warnings unfortunately don't give a hint about what went wrong :-/ Is there more on the console? Maybe above and below?

**Comment by moorebi** [comment]
Humm, I tried again here is the screen shot -- I also enabled "debug logs"  in the plugin just not sure where they are put in the folder.. If you tell me where they are located I can attached them to the this chat... 

![image](https://user-images.githubusercontent.com/6247939/166312601-62263df6-cb41-40a9-90e6-a78b3e570c86.png)

I am also experiencing an error when I click on a geo code `[Boston](geo: 42.3145186, -71.110365)` Windows 10 asks me for a new app to open the geo link.

![image](https://user-images.githubusercontent.com/6247939/166312883-694e2f2f-50fa-40ab-b8ac-fecbd1424fef.png)

**Comment by kalmir** [comment]
You're right, I was using older version. After updating and setting the zoom limit, problem solved. Thanks a lot for implementing the zoom limit option.
